module/Application/language/nl.po

```diff
@@ -8,16 +8,16 @@ msgid ""
 msgstr ""
 "Project-Id-Version: GEWISweb 0.1.0-dev\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-09-26 19:21+0200\n"
-"PO-Revision-Date: 2023-09-26 19:23+0200\n"
+"POT-Creation-Date: 2023-09-30 23:18+0200\n"
+"PO-Revision-Date: 2023-09-30 23:20+0200\n"
 "Last-Translator: Tom Udding <web@gewis.nl>\n"
 "Language-Team: English <kde-i18n-doc@kde.org>\n"
 "Language: nl\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Poedit 3.3.2\n"
+"X-Generator: Poedit 3.1.1\n"
```

Comment:
Please note that you are using an older version of Poedit, this may affect certain characters in the translation files (not the case for this PR).

Reply:
Good to know!